package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,777 @@
+            bond analysis module.
+
+            As the _timeseries to timeserie conversion will be deprecated in 1.0
+            this function will automatically lose its value.
+        """
+
+        return (list(hb[:2])
+                + [atomformat.format(hb[2]), atomformat.format(hb[3])]
+                + list(hb[4:]))
+
+    @property
+    def timeseries(self):
+        r'''Time series of water bridges.
+
+        Due to the intrinsic complexity of water bridge problem, where several
+        atoms :math:`n` can be linked by the same water. A simple ``atom 1 - water bridge - atom 2`` mapping will create a very large list
+        :math:`n!/(2(n-2)!)` due to the rule of combination.
+
+        Thus, the output is arranged based on each individual bridging water
+        allowing the later reconstruction of the water network. The hydrogen
+        bonds from selection 1 to the **first** bridging water is followed by
+        the the hydrogen bonds from selection 2 to the **same** bridging water.
+        After that, hydrogen bonds from selection 1 to the **second** bridging
+        water is succeeded by hydrogen bonds from selection 2 to the **same
+        second** bridging water. An example of the output is given in
+        :ref:`Analysis Output` .
+
+        Note
+        ----
+        To find an acceptor atom in :attr:`Universe.atoms` by
+        *index* one would use ``u.atoms[acceptor_index]``.
+
+        The :attr:`timeseries` is a managed attribute and it is generated
+        from the underlying data in :attr:`_timeseries` every time the
+        attribute is accessed. It is therefore costly to call and if
+        :attr:`timeseries` is needed repeatedly it is recommended that you
+        assign to a variable::
+
+           w = WaterBridgeAnalysis(u)
+           w.run()
+           timeseries = w.timeseries
+
+        See Also
+        --------
+        :attr:`table` : structured array of the data
+        '''
+        HydrogenBondAnalysis.timeseries(self)
+
+    def generate_table(self):
+        """Generate a normalised table of the results.
+
+        The table is stored as a :class:`numpy.recarray` in the
+        attribute :attr:`~HydrogenBondAnalysis.table`.
+
+        See Also
+        --------
+        WaterBridgeAnalysis.table
+        """
+        HydrogenBondAnalysis.generate_table(self)
+
+    def save_table(self, filename="wbridge_table.pickle"):
+        """Saves :attr:`~WaterBridgeAnalysis.table` to a pickled file.
+
+        If :attr:`~WaterBridgeAnalysis.table` does not exist yet,
+        :meth:`generate_table` is called first.
+
+        Parameters
+        ----------
+        filename : str (optional)
+             path to the filename
+
+        Example
+        -------
+        Load with ::
+
+           import cPickle
+           table = cPickle.load(open(filename))
+        """
+        HydrogenBondAnalysis.save_table(self, filename)
```

Comment:
use `super()` instead

Reply:
@orbeckst I will need to get another PR for this method. The current save_table() method won't work in py36 giving an error.
cPickle.dump(self.table, f, protocol=cPickle.HIGHEST_PROTOCOL)
TypeError: write() argument must be str, not bytes